drools-spring-app/src/main/java/sbnz/integracija/example/security/config/WebSecurityConfig.java

```diff
@@ -31,7 +31,7 @@ public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
     protected void configure(HttpSecurity http) throws Exception {
         http.csrf().disable();
         http.sessionManagement().sessionCreationPolicy(STATELESS);
-        http.authorizeRequests().antMatchers("/users/noUsr", "/books/**", "/users/register", "/login","/order/**").permitAll();
+        http.authorizeRequests().antMatchers("/users/noUsr", "/books/**", "/users/register", "/login",  "/review", "/order/**", "/recommend-unauthorized").permitAll();
```

Comment:
Can everyone rate a book/ write a review on the book? This line permits it to all users, authorized or unauthorized